definitions/infra-azureeventgridpartnertopic/definition.yml

```diff
@@ -1,5 +1,5 @@
 domain: INFRA
-type: AZUREEVENTGRIDSYSTEMTOPICS
+type: AZUREEVENTGRIDPARTNERTOPIC
```

Comment:
this one is singular, but the event in line 18 is plural. Are you sure about it?